sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -863,7 +862,7 @@ private async Task RunAsync(CancellationToken cancellationToken)
                 await Task.WhenAll(InstanceOwnership
                     .Select(async kvp =>
                     {
-                        if (!ActivePartitionProcessors.TryGetValue(kvp.Key, out var t) || t.Item1.IsCompleted)
+                        if (!ActivePartitionProcessors.TryGetValue(kvp.Key, out var activeTokenSource) || activeTokenSource.Item1.IsCompleted)
```

Comment:
I agree that `t` is not a good name, but `activeTokenSource` is not enough to describe both items of the pair (task, tokenSource). Maybe `activeTaskTokenPair` or `activeTaskToken` or `activePair` or just `pair`?

Reply:
I'll give it a bit more thought and include in the next samples PR.